test/test_autograd.py

```diff
@@ -203,6 +203,34 @@ def compute_grad(create_graph):
         x_grad, x_grad_clone = compute_grad(create_graph=True)
         self.assertEqual(x_grad, x_grad_clone)
 
+    def test_accumulate_grad_tensor_reference(self):
+        def _test_grad_tensor(params_grad_tensor, backward_grad_tensor, should_preserve_reference):
+            params = torch.tensor([1.5, 1.5]).requires_grad_()
+            with torch.no_grad():
```

Comment:
I don't think you need torch.no_grad here.

Reply:
Fixed.